src/PrestaShopBundle/Form/Admin/Sell/Product/Description/DescriptionType.php

```diff
@@ -77,6 +77,23 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $productId = (int) $options['product_id'];
+        $builder
+            ->add('packed_products', EntitySearchInputType::class, [
+                'label' => $this->trans('Pack of products', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h2',
+                'entry_type' => PackedProductType::class,
+                'entry_options' => [
+                    'block_prefix' => 'packed',
```

Comment:
Are you sure this is useful? Seems like your block name is `packed_product_row` so it seems to be using the default block prefix from `PackedProductType` Maybe the `entry_options` are not needed here

Reply:
whitout the entry_option we've got an exception like "Neither the property "id" nor one of the methods..." so twig is not able to find form.id